src/tests/Microsoft.PowerFx.Interpreter.Tests/FlagTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.Preview;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class FlagTests
+    {
+        [Theory]
+        [InlineData("Sum(A, A[@Value])")]
+        [InlineData("Filter(A, A[@Value] = 2)")]
+        public void Basic_DisableAtTableField(string expression)
+        {
+            FeatureFlags.DisableAtTableField = true;
+            var engine = new RecalcEngine(new PowerFxConfig());
+            
+            NumberValue r1 = FormulaValue.New(1);
+            NumberValue r2 = FormulaValue.New(2);
+            NumberValue r3 = FormulaValue.New(3);
+            
+            TableValue val = FormulaValue.NewSingleColumnTable(r1, r2, r3);
+
+            engine.UpdateVariable("A", val);
+            
+            var result = engine.Check(expression);
+
+            Assert.False(result.IsSuccess);
```

Comment:
>Assert.False(result.IsSuccess); [](http://example.com/codeflow?start=12&length=31)

It would be good to run the expression twice:
1. success when flag is off
2. fails when flag is on